### ISSUE TITLE
Support for new feature

### ISSUE THREAD

**Issue by juventus18** [FantasticFiasco/axis-js#671]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by FantasticFiasco** [comment]
Hi @juventus18, is this something you would like to contribute?

**Comment by juventus18** [comment]
@FantasticFiasco yeah, I think I'll do a subset of the VAPIX API and focus only on the text overlays.
